index.html

```diff
@@ -2974,134 +2983,367 @@ <h3>Obtaining a manifest</h3>
+			        either <var>manifest</var>, or an object appearing within <var>manifest</var> (e.g., a
+			            <var>Person</var>). The algorithm replaces or adds some terms to <var>manifest</var>; the
+			        replacement terms are expressed in JSON syntax as <code class="json">{"term":"value"}</code>.</p>
+			    <ol>
+			        <li>let <var>lang</var> string represent the default language, set to: <ul>
+			                <li> the value of the <a data-cite="!html#the-lang-and-xml:lang-attributes"
+			                            ><code>lang</code> value</a> for the <code>script</code> element in the
+			                        <a>primary entry page</a>, in case the manifest is <a href="#manifest-embedding"
+			                        >embedded</a>; or </li>
+			                <li><code>undefined</code> otherwise</li>
+			            </ul>
+						<details>
+							<summary>
+								Explanation
+							</summary>
+							<p>This value is used in <a href="#can-set-lang">the step on language</a> below.</p>
+						</details>
+			        </li>
+			        <li> let <var>dir</var> string represents the base direction, set to: <ul>
+			                <li> the value of the <a data-cite="!html#the-dir-attribute"><code>dir</code> value</a> for
+			                    the <code>script</code> element in the <a>primary entry page</a>, in case the manifest
+			                    is <a href="#manifest-embedding">embedded</a>; or </li>
+			                <li><code>undefined</code> otherwise</li>
+			            </ul>
+						<details>
+							<summary>
+								Explanation
+							</summary>
+							<p>This value is used in <a href="#can-set-dir">the step on base direction</a> below.</p>
+						</details>
+			        </li>
+			        <li> (<a href="#wp-title"></a>) if <var>manifest["name"]</var> is <code>undefined</code>, then
+			            locate the <a data-cite="!html#the-title-element"><code>title</code></a> HTML element using
+			                <var>document</var>. If that element exists and is non-empty, let <var>t</var> be its text content, and add to <var>manifest</var>:
+			                <ul>
+			                    <li> if the language of <code>title</code> is explicitly <a
+			                            data-cite="!html#the-lang-and-xml:lang-attributes">set</a> to the value of
+			                            <var>l</var>, then add <br />
+			                        <code class="json">"name": [{"value": t, "language": l}]</code>
+			                    </li>
+			                    <li>or <br />
+			                        <code class="json">"name": [t]</code><br /> otherwise </li>
+			                </ul>
+			                <details>
+			                    <summary>Explanation</summary>
+			                    <p>This step ensures that the <code>title</code> element's content in the primary entry page can be used, as a default, as the title of the publication. I.e.,</p>
```

Comment:
Minor nit, but what follows are examples of what is being stated, not alternate expressions, so should lead with a "for example" not "i.e."